inventory/inventory

```diff
@@ -17,7 +17,8 @@ mesos_slaves
 load_balancers
 
 [all:vars]
-ansible_python_interpreter="PATH=/home/core/bin:$PATH python"
+# The only user able to ssh to a newly created Rackspace instance is root (at least initially), so we need to update the python PATH to accommodate for that.
+ansible_python_interpreter="PATH=/home/core/bin:/root/bin:$PATH python"
```

Comment:
can we set a var group that is explicit to "rackspace" somehow, since there will be nothing in /root/bin elsewhere